### PR TITLE
Roll out scriptworker-39.0.0

### PR DESCRIPTION
**Pull request by escapewindow**
Mainly removes the old CoT keys.

### CODE REVIEW

**Comment by escapewindow** [comment]
(We might not roll this out til I'm back from PTO, but adding more verbose info just in case, + posterity.)

When we roll out, this PR will stop marking the old CoT keys as valid. In theory I got everything working, but it's this may affect repos+branches where I missed rebuilding cached tasks, or older release graphs where we want to, say, do off-cycle partner repacks where we'll depend on older tasks.

If we hit CoT errors around signatures, try to trace back to the offending taskId (should be in the log). This will likely be a cached task. Rebuilding all the cached tasks on that repo/branch/level should fix the problem, though we'll need a rebuild of the relevant shippable build(s) afterwards. To do so:

```
# find the decision task
# download full-task-graph.json
# get the labels of the cached tasks:
gron full-task-graph.json | grep cached_task | cut -d . -f 1 | sort | uniq | sed -e 's/json\["//' -e 's/"\]//' | tee cached_tasks.txt
# go back to that decision task's task group, `add-new-jobs` those labels
```

Alternately, we can back out the scriptworker 39.0.0 rollout, or alternately craft a new scriptworker release after backing out [this commit](https://github.com/mozilla-releng/scriptworker/commit/1b13b1b0628b269b7fd75f12514671bfb3143cfc), and roll that out.

Rolling out to the `production` or `production-*script` branches will update the scriptworker k8s pools.

File: addonscript/requirements/base.py37.txt [hunk 1]
```diff
@@ -180,9 +180,9 @@ jsonschema==3.2.0 \
     --hash=sha256:4e5b3cf8216f577bee9ce139cbe72eca3ea4f292ec60928ff24758ce626cd163 \
     --hash=sha256:c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a
     # via scriptworker
-jwcrypto==0.9.1 \
-    --hash=sha256:12976a09895ec0076ce17c49ab7be64d6e63bcd7fd9a773e3fedf8011537a5f6 \
-    --hash=sha256:63531529218ba9869e14ef8c9e7b516865ede3facf9b0ef3d3ba68014da211f9
+jwcrypto==1.0 \
```
Comment (escapewindow) [inline addonscript/requirements/base.py37.txt:183]: Post release bump to 1.0.dev1
    Create SECURITY.md
    Show Code Scan button and exclude trivial pulls
    Allow empty payloads in JWS tokens
    Add tests to check empty payload support
    Drop python2 compatibility
    Fix python3 pylint issues
    Add explicit support to check 'typ' in JWT
    Drop support for importing old MutableMapping
    Disable annoying pep8 naming checks
    Version 1.0
https://github.com/latchset/jwcrypto/releases/tag/v1.0.0

File: addonscript/requirements/base.py37.txt [hunk 2]
```diff
@@ -313,28 +313,27 @@ rsa==4.7.2 \
...
     #   taskcluster
 slugid==2.0.0 \
     --hash=sha256:a950d98b72691178bdd4d6c52743c4a2aa039207cf7a97d71060a111ff9ba297 \
     --hash=sha256:aec8b0e01c4ad32e38e12d609eab3ec912fd129aaf6b2ded0199b56a5f8fd67c
     # via taskcluster
-taskcluster==39.2.0 \
-    --hash=sha256:07dec0b12c80122498d772a242f92373d65731f133fb0d9aba5c2adf78475b13 \
-    --hash=sha256:844d58207f7ccba73cce2413ff79ad83ef39e4095e0269fb82ff14fa9b1c3e2f \
-    --hash=sha256:e5fe89c1f85b3caf141106053c1e98bcb930a629889e67acb2bd9aa92bd95182
+taskcluster==44.1.0 \
```
Comment (escapewindow) [inline addonscript/requirements/base.py37.txt:333]: Hmmmm, scriptworker integration tests are breaking, might be worth more investigation =\

**Comment by escapewindow** [comment]
Closing until I verify scriptworker + tc.py>39 compatibility.
We're still running tc 39.2.0 on the mac signers.
